odp_sdk/raw_storage_client.py

```diff
@@ -143,7 +144,7 @@ def upload_file(
 
         headers = {"Content-Type": "application/octet-stream"}
 
-        response = self.http_client.patch(url, headers=headers, content=contents)
+        response = self.http_client.patch(url, params={"overwrite": overwrite}, headers=headers, content=contents)
```

Comment:
Missing explanation of the overwrite on the docstring. I had to read through the code to understand it